test/unstable/serial.good

```diff
@@ -0,0 +1,203 @@
+serial.chpl:1: warning: 'serial' statements are unstable and likely to be deprecated in a future release
+serial.chpl:7: In function 'main':
+serial.chpl:8: warning: 'serial' statements are unstable and likely to be deprecated in a future release
+hello 1
```

Comment:
Consider marking the test `noexec`, particularly since the unstable nature of `serial` means the output _might_ change? Not that it's likely in such a trivial case.

Reply:
If the output changes we can update the test! Anyway with `serial` as it is described in the spec currently, it's going to have the behavior I am checking with this test. (We presumably have other tests of `serial` in this way, but I thought it very easy to test it here as well).

Reply:
Fine by me 🙂